src/OSPSuite.Presentation/Presenters/Charts/ChartDisplayPresenter.cs

```diff
@@ -497,22 +491,7 @@ private void updateYAxisVisibility(AxisTypes yAxisType)
          axisBinder.Visible = axisTypeUsed;
       }
 
-      private void setDisplay(DockStyle dockStyle, ChartFontAndSizeSettings fontAndSizeSettings, bool previewOriginText)
```

Comment:
There was a lot of back and forth between presenter and view. I streamlined the code a bit